units/en/unit5/introduction.mdx

```diff
@@ -0,0 +1,45 @@
+# Introduction
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://githubtocolab.com/dylanebert/ml-for-3d-course-notebooks/blob/main/capstone.ipynb)
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/l-KcMssIhI4?si=bAsM1wYWWenJjoAV" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+Welcome to the final capstone project of the course.
+
+To complete the course and receive a certificate, you will create and host your own Generative 3D model and demo.
+
+## Requirements
+
+- **Model**: A model hosted under your username, e.g. `hf.co/{username}/{model_name}`.
+  - Ensure there is a [model card](https://huggingface.co/docs/hub/en/model-cards).
+  - Tag the model with the `image-to-3d` pipeline tag.
+  - Apply the correct license if applicable.
+  
+- **Space**: A space hosted under your username, e.g. `hf.co/spaces/{username}/{space_name}`.
+  - Ensure the space converts images (`.png`, `.jpg`) to 3D (`.glb`, `.obj`, `.ply`, `.splat`).
+  - Reference your model URL in the space README.
+
+When completed, fill out this [form](https://forms.gle/rQedXFktHPYeikrt6) to receive your certificate.
+
```

Comment:
If we can give an explicit reference example or two here so that people can try it out, note: a lot of people would do what you show here i.e. the minimum requirements so it's good to leave it just enough open-ended that there's some creativity but not too open-ended in the sense that they take the easiest way out.

Reply:
So a good example would be to make a pipeline with text to image to 3D or other things where people can sequentially stack a couple of different models/APIs together.